modules/swagger-parser/src/main/java/io/swagger/parser/ResolverCache.java

```diff
@@ -66,6 +66,27 @@ public ResolverCache(Swagger swagger, List<AuthorizationValue> auths, String par
         this.swagger = swagger;
```

Comment:
instead of duplicating logic in constructor I would suggest to call `this(swagger, auths, parentFileLocation, new ParseOptions())`